fluffy.py

```diff
@@ -40,7 +40,7 @@ def article(topic=None, company=None):
     ))
     company = company or "ACME"
 
-    yield from intoduction(topic)
+    yield from introduction(topic, company)
```

Comment:
I do too, but filling in "ACME" in the argument space generates the same error, Are there arg-parsing issues?